app/views/idv/doc_auth/front_image.html.erb

```diff
@@ -0,0 +1,44 @@
+<% title t('doc_auth.titles.doc_auth') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render 'idv/doc_auth/front_of_state_id_image' %>
+
+<h1 class='h3 my0'>
+  <%= t('doc_auth.headings.upload_front') %>
+</h1>
+
+<%= accordion('totp-info', t('doc_auth.tips.title_html'),
+  wrapper_css: 'my2 col-12 fs-16p') do %>
+  <%= render 'idv/doc_auth/tips_and_sample' %>
+  <div class='center'>
+    <%= image_tag(asset_url('state-id-sample-front.jpg'), height: 338, width: 450) %>
+  </div>
+<% end %>
+
+<%= simple_form_for(
+  :doc_auth,
+  url: url_for,
+  method: 'PUT',
+  html: { autocomplete: 'off', role: 'form', class: 'mt2' }
+) do |f| %>
+  <%= f.input :image_data_url, as: :hidden %>
+  <div id='acuant-fallback-image-form'>
+    <%= f.input :image, label: false, as: 'file', required: true, wrapper_class: 'mt3 sm-col-8' %>
+    <div class='my2' id='target'></div>
+    <div class='mt3'>
+      <%= render 'idv/doc_auth/submit_with_spinner' %>
+    </div>
+  </div>
+  <div class='my3'>
```

Comment:
I added this new div which contains a partial to init the Acuant SDK and has the controls the SDK needs.